tests/README.md

```diff
@@ -94,20 +100,22 @@ If you wish to run these tests locally, skip ahead to
    following key/values:
    - AZDO_PROJECT (e.g. `bedrock`)
    - AZDO_ORG (e.g. `epicstuff`)
-   - AZDO_PAT (e.g. Personal Access Token with _read/write/manage_ access to
+   - AZDO*PAT (e.g. Personal Access Token with \_read/write/manage* access to
```

Comment:
Not sure what these changes are adding.

Reply:
If this is caused by husky, might be good to disable temporarily.

Reply:
Resolved